p/themes/base-theme/frss.rtl.css

```
@@ -1,4 +1,5 @@
 @charset "UTF-8";
+@import url('https://fonts.googleapis.com/css2?family=Vazirmatn:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Same, but with something like:

```suggestion
/*rtl:raw:@import url('https://fonts.googleapis.com/css2?family=Vazirmatn:wght@100;200;300;400;500;600;700;800;900&display=swap');*/
```

---

I would rather avoid a query to a Google server if possible. Could this fond be exported to be used locally?

---

Yes it could be used locally. But first i need to look into themes structure so i can implement it better...

---

have a look into the CSS `@font-face`.
Font files are stored in the directory `/p/themes/fonts`